server/app.js

```diff
@@ -1,17 +1,24 @@
-const express = require('express')
+const fs = require('fs')
 const path = require('path')
+const express = require('express')
 const cors = require('cors')
 const router = require('./routes.js')
 const logger = require('morgan')
 const models = require('./models')
-const fs = require('fs')
 const multer = require('multer')
+const AWS = require('aws-sdk')
 
 const app = express()
 
-const PHOTO_UPLOAD_FOLDER = path.join('/Users/louisa/HR/HR2021/SDC/Atelier', '/client/UploadedPhotos');
 
-app.use(express.static(PHOTO_UPLOAD_FOLDER));
+const staticAssets = path.resolve(__dirname.substring(0, __dirname.length - 17), 'Atelier/client/dist')
+app.use(express.static(staticAssets))
+
+
+const PHOTO_UPLOAD_FOLDER = path.join(__dirname.substring(0, __dirname.length - 17), 'Atelier/client/UploadedPhotos');
+app.use(express.static(PHOTO_UPLOAD_FOLDER))
```

Comment:
On the front end I noticed that the FEC team had a photo folder so I just also re-directed my photo uploads to this folder as well. Not a good idea to keep photos in the backend anyways due to greater potential for bloating and slowing down the db process. This method is a lot faster for client-side.